src/plugins/memdump/memdump.cpp

```diff
@@ -873,6 +874,167 @@ static event_response_t set_information_thread_hook_cb(drakvuf_t drakvuf, drakvu
     return VMI_EVENT_RESPONSE_NONE;
 }
 
+struct exec_fault_data
+{
+    memdump* plugin;
+    addr_t rip;
```

Comment:
Why is this variable named rip? I understand it as the starting virtual address of the page in which some execution will fault in the future?

Reply:
yes

Reply:
I will modify it to some better name like page_va

Reply:
You could do this at the ipt plugin as well